requirements.txt

```diff
@@ -13,6 +13,6 @@
 #   limitations under the License.
 
 pika==1.2.0
-th2-grpc-common==3.3.0
-kubernetes==17.17.0
+th2-grpc-common==3.4.0
+kubernetes==18.20.0
```

Comment:
Why do we update the Kubernetes version?

Reply:
Dependabot notified about dependency update earlier.
New version doesn't break the compatibility, so there should be no problems.